### PR TITLE
update dependencies from setup.py of v1.6.1

### PR DESCRIPTION
**Pull request by manuelwalserqc**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by manuelwalserqc** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,20 +24,23 @@ requirements:
...
+    - click >=7.1.1,<8
+    - deprecated >=1.2.7,<2
+    - strip-hints >=0.1.8,<1
+    - docstring_parser >=0.7.3,<1
+    - requests-toolbelt >=0.8.0,<1
+    - fire >=0.3.1,<1
+    - protobuf >=3.13.0,<4
     - absl-py >=0.9,<=0.11
 
 test:
```
Comment (xhochy) [inline recipe/meta.yaml:46]: Can you please add a check that the dependencies match?
```suggestion
test:
    requires:
      - pip
    commands:
      - pip check
```

**Comment by maximerihouey** [comment]
@manuelwalserqc Thank you for updating the dependencies and making the CI pass. I preferred to use the original MR opened by Conda-Forge so I updated it with your contribution: https://github.com/conda-forge/kfp-feedstock/pull/45
I also added you as a maintainer, we can close this MR.

@xhochy Thank you for the suggestion, I added it to https://github.com/conda-forge/kfp-feedstock/pull/45

**Comment by xhochy** [comment]
> @manuelwalserqc Thank you for updating the dependencies and making the CI pass. I preferred to use the original MR opened by Conda-Forge so I updated it with your contribution: https://github.com/conda-forge/kfp-feedstock/pull/45

What is the background here to not merge contributor-created PRs? This sounds a bit weird.

**Comment by maximerihouey** [comment]
> > @manuelwalserqc Thank you for updating the dependencies and making the CI pass. I preferred to use the original MR opened by Conda-Forge so I updated it with your contribution: #45
> 
> What is the background here to not merge contributor-created PRs? This sounds a bit weird.

PRs created by Conda-Forge are all formatted the same.
I see that you keep closing PR, this is even weirder.

EDIT: I guess the logic behind closing the PRs is that there are no corresponding TAG in the `kubeflow/pipelines` for version `1.6.2`,  `1.6.3`, etc... Even `1.6.1` actually.

**Pull request closed without merging**